gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -226,17 +226,19 @@ <h3 class="mdc-layout-grid__cell--span-3">
                       {% set link = event | event_link -%}
                       {% if link -%}
                         <a href="{{ link }}">
+                          {{ event | event_value -}}
+                        </a>
                       {% elif event | event_type == 'Introduced' and event | event_value == '0' -%}
                         <div class="tooltip">
-                      {% endif -%}
-
-                      {{ event | event_value -}}
-
-                      {% if link -%}
-                        </a>
-                      {% elif event.get('introduced') == '0' -%}
-                        <span class="tooltiptext">The exact introduced commit is unknown</span>
+                          {{ event | event_value -}}
+                          {% if range.type == 'GIT' %}
+                            <span class="tooltiptext">Unknown commit / All previous commits are affected</span>
+                          {% else -%}
+                            <span class="tooltiptext">Unknown version / All previous versions are affected</span>
```

Comment:
I slightly updated the suggested text for the tooltip to get it fit a single line. With long messages the introduced and fixed rows will be one line apart:
<img width="1018" alt="Screenshot 2024-06-25 at 12 17 20 PM" src="https://github.com/google/osv.dev/assets/73332835/9d580221-4727-407f-a670-ad0c5b7774dd">